woocommerce/wc-admin
# Payments task: include Mercado Pago

Fixes #6537

This PR adds Mercado Pago as a payment method.

### Accessibility




### Screenshots
![screenshot-one wordpress test-2021 03 12-10_57_36](https://user-images.githubusercontent.com/1314156/110949852-dd023180-8321-11eb-9ec3-8f368a35db97.png)

![screenshot-one wordpress test-2021 03 12-10_13_01](https://user-images.githubusercontent.com/1314156/110945274-3cf5d980-831c-11eb-9af8-77bc0c58bebc.png)


### Detailed test instructions:
_Be sure to be using the latest WC (`trunk` branch) version in order to have the `Mercado Pago` logo available._

- Create a brand new store.
- Set one of the following countries in the first OBW step:
```
Mexico
Brazil
Argentina
Chile
Colombia
Peru
Uruguay
```
- Continue with the OBW and finish it up. 
- Select `Choose payment methods` in the setup task list (`Get ready to start selling`).
- Press the `Setup` button in the `Mercado Pago Payments` box.
- Try the links presented after the plugin's installation and verify they are working.
- Confirm that the `Mercado Pago payments for WooCommerce` plugin was installed.
- Press `Continue`.
- Now the `Mercado Pago Payments` option should appear as active.
